RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -118,6 +131,59 @@
     estimator = 'jetCoreRegionalStepChi2Est',
     maxDPhiForLooperReconstruction = cms.double(2.0),
     maxPtForLooperReconstruction = cms.double(0.7)
+    )
+seedingDeepCore.toModify(jetCoreRegionalStepTrajectoryBuilder,
+    maxPtForLooperReconstruction = cms.double(0),
+    keepOriginalIfRebuildFails = True,
+    lockHits = False,
+    requireSeedHitsInRebuild = False,
+    trajectoryFilter = cms.PSet(refToPSet_ = cms.string('CkfBaseTrajectoryFilter_blockLoose'))
+)
+
+#customized cleaner for DeepCore
+trajectoryCleanerBySharedHits_JetCore = cms.ESProducer("TrajectoryCleanerESProducer",
```

Comment:
from a git grep all other patterns prefix the iteration info in the name; it's also in the ComponentName;
they also clone from existing object and correspondingly update only needed values instead of a definition from scratch.

This comment applies to a few more patterns in this file

Reply:
ok fixed name and cloned from existing object both this and the trajectoryFilter, in the next commit